### ISSUE TITLE
Combination with external resource & inline js

### ISSUE THREAD

**Issue by in-session** [DirkPersky/typo3-dp_cookieconsent#12]
Could it be that a combination of the both is not possible? 

```
<script type="text/plain" data-cookieconsent="statistics" data-src="https://s3.tradingview.com/external-embedding/embed-widget-tickers.js">
  {
  "symbols": [
    {
      "title": "GOLD (USD)",
      "proName": "XAUUSD"
    }
  ],
  "colorTheme": "dark",
  "locale": "en"
}
</script> 
```

**Comment by DirkPersky** [comment]
Yeah that ist right.

**Comment by mario-seidel** [comment]
I already tested the combination of internal js with an external source. And with this, you could first load the resource and then add the inner HTML as callback, which will be called if source was loaded.
But imho this is not a good approach, because its a little misuse of the script tag.
 
Another option is to add an ID to the script element created when cookies are accepted so that you can add an event listener to that element. On the other hand, you depend on the build and can get an error if the item is not available.

In summary, a good solution for the existing callback function after loading the exteral script would be really nice.

**Comment by mario-seidel** [comment]
:+1:
